src/main/java/com/launchdarkly/openfeature/serverprovider/Version.java

```diff
@@ -4,5 +4,5 @@ abstract class Version {
   private Version() {}
   
   // This constant is updated automatically by our Gradle script during a release, if the project version has changed
-  static final String SDK_VERSION = "0.2.0";
+  static final String SDK_VERSION = "1.0.1";
```

Comment:
Hmm. I wonder if there is a gradle problem here. Doesn't need to be sorted in this PR.